pkg/auth/ldaputil/query.go

```diff
@@ -137,6 +149,11 @@ func (o *LDAPQueryOnAttribute) buildAttributeQuery(attributeValue string,
 
 	filter := fmt.Sprintf("(&(%s)(%s))", o.Filter, specificFilter)
```

Comment:
It seems like we are, but the double-parens are tolerated? Our test sync configs in extended test declare filters with parens and they run fine.